### DISCUSSION TITLE
Need to disable mouse wheel scrolling on a Carousel control

### DISCUSSION THREAD

**Discussion by ionite34** [AvaloniaUI/Avalonia#12655] [Q&A]
I'm currently making a Carousel control that has a zoom/pan enabled image viewer inside, the issue is that the Carousel and my image viewer is capturing the pointer wheel events at the same time.

How should I disable any mouse wheel interactions with the Carousel? I've tried the following, but this also prevents my image viewer from capturing the scroll events.
```xml
    <Style Selector="Carousel /template/ ScrollViewer#PART_ScrollViewer">
        <Setter Property="IsEnabled" Value="False"/>
    </Style>
```

**Comment by timunie** [comment]
I guess you are on the right track. Set SctollViewer.VerticalScrollbarVisibility to Disabled
